sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -293,10 +293,11 @@ export class SenderImpl implements Sender {
     return retry<void>(config);
   }
 
-  async open(options?: SenderOpenOptions): Promise<void> {
+  async open(options?: OperationOptionsBase): Promise<void> {
     this._throwIfSenderOrConnectionClosed();
 
     const config: RetryConfig<void> = {
+      // TODO: Pass tracing options too
```

Comment:
Log an issue for this please?

Reply:
Yes!!

Reply:
Logged an issue https://github.com/Azure/azure-sdk-for-js/issues/10637

Also, not just this, we aren't handling tracing options in any of the methods based on AMQP operations.